products/compute/terraform.yaml

```diff
@@ -630,17 +606,12 @@ overrides: !ruby/object:Provider::ResourceOverrides
         custom_flatten: templates/terraform/custom_flatten/name_from_self_link.erb
```

Comment:
I noticed a line above here that kms key is excluded because it showed up in GA, but it appears to be marked [`beta` in api.yaml](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/products/compute/api.yaml#L3160-L3161) do you know why it's showing up in GA?

Reply:
I suspect because it's a nested field. I don't think this PR is the right place to solve that - we can do it in a followup.

Reply:
Bummer, I assume you filed a bug on the docs?